ejb3/src/main/java/org/jboss/as/ejb3/remote/AssociationImpl.java

```diff
@@ -217,6 +230,34 @@ public CancelHandle receiveInvocationRequest(@NotNull final InvocationRequest in
         return cancellationFlag::cancel;
     }
 
+    private static <A> boolean intersect(final Set<A> fst, final Set<A> snd) {
+        fst.retainAll(snd);
+        return !fst.isEmpty();
+    }
+
+    private static Set<String> typesOf(final Object o) {
```

Comment:
Right but I'm thinking of the fix you did for EAP 7.0.x.  Did any problems like this come up during testing?